src/shared/components/plots/MultipleCategoryBarPlotUtils.ts

```diff
@@ -121,20 +131,114 @@ export function makeBarSpecs(
 }[] {
     // one bar spec per minor category, in correct order - either specified, or alphabetical
     data = sortDataByCategory(data, d => d.minorCategory, minorCategoryOrder);
+
     // reverse the order of stacked or horizontal bars
     if ((!horizontalBars && stacked) || (horizontalBars && !stacked)) {
         data = _.reverse(data);
     }
+    console.log(data, 'iscalled2');
+    data.forEach(item => {
```

Comment:
this whole logic looks similar to the one above https://github.com/cBioPortal/cbioportal-frontend/blob/a05006df4870e64bd5df1dec6a485d3da8ca237d/src/pages/groupComparison/MultipleCategoryBarPlot.tsx#L440

would be great if we could make a helper function with all this logic to minimize duplicate code.